platform/.eslintrc.json

```diff
@@ -26,13 +26,16 @@
       "error",
       {
         "zones": [
-          { "target": "./src/core", "from": "./src/components" },
+          { "target": "./src/core", "from": "./src/common" },
           { "target": "./src/core", "from": "./src/lib" },
           { "target": "./src/core", "from": "./src/pages" },
           { "target": "./src/lib", "from": "./src/pages" },
-          { "target": "./src/components", "from": "./src/pages" }
+          { "target": "./src/common", "from": "./src/pages" }
         ]
       }
-    ]
+    ],
+    "import/prefer-default-export": "off",
```

Comment:
We use both default and named exports, however, eslint rules to use default exports only so I turned that rule off to enable support for named exports.